e2e/features/issue-decision.feature

```diff
@@ -15,7 +15,7 @@ Feature: Issue decision
     And I navigate to an existing case
     When I choose "Write final decision"
 
-  @issue-decision @nightly-test
+  @issue-decision @nightly-test @crossbrowser99
```

Comment:
@phaniperla  should this be @crossbrowser ?

Reply:
There is a known issue in sauce labs wrt to remote file uploading and no workaround solution available so therefore that scenario has been excluded from  cross browser.

Reply:
Please refer below JIRA ticket for further details

Reply:
https://tools.hmcts.net/jira/browse/TA-636